bin/product-type-export

```diff
@@ -5,6 +5,8 @@ var ProjectCredentialsConfig = require('sphere-node-utils').ProjectCredentialsCo
 var ProductTypeExport = require('../dist').default
 var argv = require('minimist')(process.argv.slice(2))
 
+var enabledFormats = ['csv', 'xlsx']
```

Comment:
Updated here 4b49bb3 - it should be unified across whole project so now we have `supportedFormats` everywhere